### PR TITLE
v2.9.0 rc-1

### PR DESCRIPTION
**Pull request by brouillette**

### CODE REVIEW

**Comment by davidlougheed** [comment]
if it's not used anywhere anymore, it'd be good to remove all the common image stuff

File: lib/federation/docker-compose.federation.yaml [hunk 1]
```diff
@@ -3,13 +3,6 @@ version: '3.7'
...
-    build:
-      context: $PWD/lib/federation
-      args:
-        BASE_IMAGE: ${BENTOV2_FEDERATION_BASE_IMAGE}
-        BASE_IMAGE_VERSION: ${BENTOV2_FEDERATION_BASE_IMAGE_VERSION}
-        REPO: ${BENTO_FEDERATION_REPO}
-        TAG_OR_BRANCH: ${BENTOV2_FEDERATION_TAG_OR_BRANCH}
     image: ${BENTOV2_FEDERATION_IMAGE}:${BENTOV2_FEDERATION_VERSION}
     container_name: ${BENTOV2_FEDERATION_CONTAINER_NAME}
     environment:
```
Comment (davidlougheed) [inline lib/federation/docker-compose.federation.yaml:8]: `CHORD_REGISTRY_URL`, `BENTO_FEDERATION_MODE`, `INITIALIZE_IMMEDIATELY`, and `DATABASE` all can be removed from environment.

```
volumes:
      - ${BENTOV2_FEDERATION_PROD_VOL_DIR}:/federation/data
```

can be removed.

**Review by davidlougheed — CHANGES_REQUESTED** [review]

File: lib/federation/docker-compose.federation.yaml (hunk 1 above)
Comment (brouillette) [reply lib/federation/docker-compose.federation.yaml:8]: done

File: docker-compose.dev.yaml [hunk 2]
```diff
@@ -127,9 +125,7 @@ services:
       - "${BENTOV2_WES_DEBUGGER_EXTERNAL_PORT}:${BENTOV2_WES_DEBUGGER_INTERNAL_PORT}"
 
   drs:
-    build:
-      context: $PWD/lib/drs
-      dockerfile: Dockerfile.dev
+    image: ${BENTOV2_DRS_IMAGE}:latest-dev
```
Comment (davidlougheed) [inline docker-compose.dev.yaml:128]: none of the other services have the `:latest-dev` tag explicitly here; what's the plan for consistency / development?

Comment (brouillette) [reply docker-compose.dev.yaml:128]: no plan yet - this was meant as a workaround initially, but we can continue with it if it works!

**Review by davidlougheed — APPROVED** [review]

**Pull request merged**